### PR TITLE
D072896 am rm pd

### PR DESCRIPTION
**Pull request by geisslet**
removal of the old pd instance to avoid notification fails after August 1st - removal of secret keys follows separately .

+ some minor sre routing

### CODE REVIEW

File: global/prometheus-alertmanager-operated/values.yaml [hunk 1]
```diff
@@ -28,13 +28,6 @@ slack:
   vmware: {}
 
 # Values for PagerDuty receivers.
-pagerduty:
```
Comment (auhlig) [inline global/prometheus-alertmanager-operated/values.yaml:31]: I see `.Values.pagerduty.*` still being used in various places, for instance here: https://github.com/sapcc/helm-charts/blob/2db664cd2ac9e95cfc293a52750567e634139912/global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl#L759

File: global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl [hunk 2]
```diff
@@ -98,13 +98,13 @@ route:
     match_re:
       tier: os
       severity: info|warning|critical
-      service: arc|backup|barbican|castellum|cinder|cfm|designate|elektra|elk|glance|hermes|ironic|keppel|keystone|limes|lyra|maia|manila|neutron|nova|octavia|sentry|swift
+      service: arc|backup|barbican|castellum|cinder|cfm|designate|elektra|elk|glance|hermes|ironic|keppel|keystone|limes|lyra|maia|manila|neutron|nova($|-.*)|octavia|sentry|swift
```
Comment (auhlig) [inline global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Why is that needed?

Comment (joker-official) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: So SRE alerts für `nova-api` and `nova-console` and `nova-placement-api` get routed to `#cc-os-nova`, too.

Comment (auhlig) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: I see. Please be explicit in the config making this `nova|nova-api|nova-console|nova-placement-api`.

Comment (joker-official) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Why? Everything starting with nova- should go to cc-os-nova. Why do we have to search for all possible services and always update this rule if we add one?

Comment (auhlig) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Because we started with regexes a while ago which proved to become messy over time. Thus I want to keep the configuration explicit.

Comment (joker-official) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Then I vote for removing this and trying to relabel the "service" to nova in the alerts.

Comment (joker-official) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Or how about setting the `($|-.*)` at the end, so that it's valid for all services. Would that work for you? So that `arc-api` ingress also gets into the `arc` channel etc.

Comment (auhlig) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Would also work. You could still use e.g. the `context` label for logical grouping of alerts.

Comment (auhlig) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: Please no regex-ing here anymore. I've spent too many hours chasing alerts.

Comment (joker-official) [reply global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl:101]: ok then, removal it is. @geisslet could you remove that, please? We will have to put the regexes into the Alert-Definition instead.

File: global/prometheus-alertmanager-operated/values.yaml (hunk 1 above)
Comment (geisslet) [reply global/prometheus-alertmanager-operated/values.yaml:31]: Don't get it. 
Of course `.Values.pagerduty_sap.` is still in :: just `.Values.pagerduty.` is removed (and  I can't find any anymore). 
Do I miss something?

Comment (auhlig) [reply global/prometheus-alertmanager-operated/values.yaml:31]: I though I saw 2 occurrences of the old `.Values.pagerduty.`. Nevermind they seem to be gone
